main/precompiled/end.zkasm

```diff
@@ -4,13 +4,11 @@ preEnd:
     $ => SP         :MLOAD(lastSP)
     $ => PC         :MLOAD(lastPC)
     1               :MSTORE(SP++)
-    ${checkpoint()}
                     :JMP(readCode)
 
 preEndFail:
     $ => GAS        :MLOAD(gasCTX)
     $ => SP         :MLOAD(lastSP)
     $ => PC         :MLOAD(lastPC)
     0               :MSTORE(SP++)
-    ${revert()}
```

Comment:
Why don't call revertTouched?

Reply:
`precompiled` could not add any touched address so it is not need to revert anything since the touched SR will be the same